Aptoide/appcoins-wallet-android
# [bugfix/APPC-1212] poa final animation not correct

**What does this PR do?**

   This PR changes the wallet validation success animation to use one that doesn't feels like the user is receiving something that it's not 

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] ValidationSuccessFragment.kt

**How should this be manually tested?**
Start the poa with a new campagin for a new wallet for a new phone number. after submitting the code the animation should only be a check 

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1212](https://aptoide.atlassian.net/browse/APPC-1212)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1212](https://aptoide.atlassian.net/browse/APPC-1212)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
